src/modules/material-requests/dto/material-requests.dto.ts

```diff
@@ -32,7 +66,7 @@ export class MaterialRequestsQueryDto {
 		type: String,
 		description: 'property to sort the results by',
 		default: 'createdAt',
-		enum: ['id', 'objectSchemaIdentifier', 'profileId', 'reason', 'createdAt', 'updatedAt'],
+		enum: ['id', 'requesterName', 'requesterEmail', 'maintainer', 'createdAt', 'type'],
```

Comment:
optional: maybe add an enum for this and use it in places instead of keyof MaterialRequest

Reply:
Adjusted